rbac/common/role/imports_role.py

```diff
@@ -17,6 +17,7 @@
 
 import logging
 from rbac.common import addresser
+from rbac.common.protobuf import role_transaction_pb2  # pylint: disable=unused-import
```

Comment:
Is this being used?

Reply:
Yes, it’s a artifact of dynamically discovering the protobuf on a separate thread.